src/Selenium2Library/keywords/_alert.py

```diff
@@ -1,3 +1,4 @@
+from time import sleep
 from selenium.common.exceptions import WebDriverException

```

Comment:
Should have an empty like after between different import groups. See pep-8.

Also, `sleep` is a pretty generic function name. `import time` and using `time.sleep()` might be a bit clearer.